spec/models/spree/payment_spec.rb

```diff
@@ -401,6 +401,25 @@
             expect(offsetting_payment.response_code).to eq('12345')
             expect(offsetting_payment.source).to eq(payment)
           end
+
+          context 'and the source payment card is expired' do
+            let(:card) do
+              Spree::CreditCard.new(month: 12, year: 1995, number: '4111111111111111')
+            end
+
+            let(:successful_response) do
+              ActiveMerchant::Billing::Response.new(true, "Yay!")
+            end
+
+            it 'lets the new payment to be saved' do
```

Comment:
I think the correct English is "it 'lets the new payment be saved'" or ""it 'allows the new payment to be saved'""